test/react-native-cli/features/steps/steps.rb

```diff
@@ -85,6 +85,8 @@ def parse_package_json
   # once and sometimes appear twice, depending on if another command is running
   # when it's input
   json = after.drop_while { |line| line.include?('cat package.json') }
+  # Drop lines until we get to the start of the JSON
+  json = after.drop_while { |line| line != '{' }
```

Comment:
We don't actually need the other `drop_while` since this line will also remove the "cat package.json" lines too — sorry if that wasn't clear